cocotb/handle.py

```diff
@@ -286,10 +296,11 @@ def __getattr__(self, name):
 
         raise AttributeError("%s contains no object named %s" % (self._name, name))
 
-    def _id(self, name, extended=True):
-        """Query the simulator for a object with the specified name,
-        including extended identifiers,
+    def _id(self, name, extended : bool = True):
+        """Query the simulator for an object with the specified *name*,
         and cache the result to build a tree of objects.
+
+        If *extended* is ``True``, run the query only for extended identifiers.
```

Comment:
After doing a bit more reading, `extended` is exclusively a VHDL feature, in verilog there is no such concept, only an escape sequence which makes `\foo<space>` and `foo` equivalent.